salt/py26-compat-salt.spec

```diff
@@ -456,6 +458,11 @@ cp %{S:5} ./.travis.yml
 %{__python} setup.py --with-salt-version=%{version} --salt-transport=both build
 cp ./build/lib/salt/_version.py ./salt
 
+# Preparations for Tornado version that's needed in the thin file
+tar xfv %{S:7}
+mkdir -p %{buildroot}/opt/tornado/lib/python2.7/site-packages/
```

Comment:
Hmm, why not using the compat dir? I think /opt is always wrong for distributors as we are.

Reply:
and probably the `tornado` part from the path is not needed

Reply:
`compatdir` also seems to be the wrong location, since tornado ships with an `.so` file. Maybe `/usr/lib`?